packages/swap/src/routes/swap.ts

```diff
@@ -132,6 +133,20 @@ router.get(
       );
     }
 
+    let depositChannelExpiryTime = new Date();
+    if (swapDepositChannel && !swapDepositChannel.isExpired) {
```

Comment:
if it expired, I'd expect to receive the time it expired at or nothing I think

Reply:
As the time long in the past may be too inaccurate, will send null for expired ones